src/actions/index.js

```diff
@@ -11,8 +11,17 @@ export const DECREMENT = 'DECREMENT';
 
 export const increment = () => {
   // Fill in this function
+  return {
+  type: INCREMENT
+  // payload: payload
+  }
 };
 
 export const decrement = () => {
   // Fill in this function
+  return{
+  type: DECREMENT
+  // payload: newPayload
+  // payload: payload
```

Comment:
Well done, notice we don't have to add a payload, only the type is required in our action